packages/AlertDialog/index.tsx

```diff
@@ -0,0 +1,161 @@
+import React, {useState} from 'react';
+import Modal, {ModalProps} from 'react-native-modalbox';
+import styled from '@emotion/native';
+
+import {StyleProp, ViewStyle, TextStyle} from 'react-native';
+
+const ModalButton = styled.TouchableOpacity<ButtonTypeProps>`
+  width: 100%;
+  height: 40px;
+  border: 1px solid black;
+  border-radius: 20px;
+  justify-content: center;
+  align-items: center;
+  background-color: ${(props) =>
+    props.isAdditional ? 'transparent' : 'black'};
+  margin-bottom: ${(props) => (props.isAdditional ? '16px' : '0px')}; ;
+`;
+
+const ButtonText = styled.Text<ButtonTypeProps>`
+  font-size: 14px;
+  font-weight: bold;
+  color: ${(props) => (props.isAdditional ? 'black' : 'white')}; ;
+`;
+
+const ModalInput = styled.TextInput`
+  height: 40px;
+  margin-bottom: 16px;
+  padding: 4px;
+  font-size: 14px;
+  border: 1px solid black;
+`;
+
+const Title = styled.Text`
+  font-size: 17px;
+  line-height: 20px;
+  text-align: center;
+  margin-bottom: 16px;
+`;
+
+const Content = styled.Text`
+  font-size: 16px;
+  line-height: 20px;
+  margin-bottom: 16px;
+`;
+
+type Styles = {
+  modal?: ModalProps['style'];
+  button?: StyleProp<ViewStyle>;
+  buttonText?: StyleProp<TextStyle>;
+};
+
+interface ButtonTypeProps {
+  isAdditional?: boolean;
+}
+
+interface Props {
+  isOpen?: boolean;
+  type?: 'alert' | 'confirm' | 'prompt';
+  title?: string;
+  content?: string;
```

Comment:
I remember you originally decided to get `content` as `children: ReactElement`.
Is there something that changes your mind?

Reply:
[Figma Design](https://www.figma.com/file/LJxf5ed71UWgUgXhA0pTtm/Dooboo-UI-powered-by-ARTIFACTS?node-id=0%3A1) existed and reflected this.